src/views/DesignAssistantSurvey.js

```diff
@@ -445,234 +461,299 @@ class DesignAssistantSurvey extends Component {
 
   render() {
     var number = 1;
-    return this.state.model ? (
+    return (this.state.model || !this.state.userQuestionAnswered) ? (
       <div>
-        <div className="dimensionNav">
-          <Accordion>
-            {this.state.dimArray.map((dimension, index) => {
-              return (
-                <Card key={index}>
-                  <Accordion.Toggle as={Card.Header} eventKey={index + 1}>
-                    {dimension}
+        {!this.state.userQuestionAnswered ? (
+          <div style={{ padding: '40px' }}>
+            <p style={{ paddingTop: '20px' }}>
+              Describe the user (person who operates the AI system) and the data
+              subject (person whose data is processed by the system). Example:
+              health care worker (user) operates system that helps diagnose
+              patient (data subject).
+            </p>
+            <fieldset id="userQuestion">
+              <div>
+                <input
+                  type="radio"
+                  value="userNotData"
+                  name="userQuestion"
+                  onChange={() => this.updateUserAnswer('userNotData')}
```

Comment:
Instead of passing the text 'userNotData', pass the event and then get the value via `event.target.value`